pkg/restic/common.go

```diff
@@ -95,13 +95,17 @@ func getPodSnapshotAnnotations(obj metav1.Object) map[string]string {
 	return res
 }
 
+func isPVBMatchPod(pvb *velerov1api.PodVolumeBackup, pod metav1.Object) bool {
```

Comment:
@ashish-amarnath if you prefer to leave it like this I'd defer to you, but it'd be preferable if you passed 4 strings (pvb name and ns, pod name and ns) instead of 2 full objects into this function.

Reply:
The PVB is passed as a pointer. The `metav1.Object` shouldn't be that bad.
So I would prefer it this way.
Marking as resolved. Please unresolved if not ok. 🙂